app/common/cluster_service.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
This whole class is just a copy/paste of a method that was previously in ClusterSlave.